apps/patient/src/views/ReadyBy.tsx

```diff
@@ -11,7 +11,8 @@ import { PoweredBy } from '../components/PoweredBy';
 import t from '../utils/text.json';
 import { OrderContext } from './Main';
 
-const currentTimeIsAfterOption = (option: string): boolean => {
+const checkDisabled = (option: string): boolean => {
+  if (option === 'tomorrow') return false;
```

Comment:
will this catch if the text is "Tomorrow"? is it lowercased somewhere?

Reply:
ah interesting, works without upper case, removing